src/commands/update/ip-blocks/put_ipblock_tag.go

```diff
@@ -0,0 +1,53 @@
+package ip_blocks
+
+import (
+	"github.com/spf13/cobra"
+	"phoenixnap.com/pnapctl/common/client/ip"
+	"phoenixnap.com/pnapctl/common/models/ipmodels"
+	"phoenixnap.com/pnapctl/common/printer"
+	"phoenixnap.com/pnapctl/common/utils"
+)
+
+var Filename string
+
+const commandName = "put ip-block tag"
```

Comment:
I'm now realizing - this file should be nested in a `tags` folder. As it is now the command will be `put ip-block` which is not correct.

Reply:
Moved this and put_ipblocktag_test to a new tags folder under update

Reply:
no - it needs to be a tags folder under `ip-blocks/tags` - and in `ip-blocks/` you need to add a command that adds this one.